v0/destinations/woopra/utils.js

```diff
@@ -0,0 +1,30 @@
+const { set } = require("lodash");
+const { getBrowserInfo } = require("../../util");
+/**
+ *
+ * @param {*} object
+ * concatenates only values of the object
+ * @returns the concatenated string
+ */
+const mergeObjectValues = object => {
+  let res;
+  Object.keys(object).forEach(v => {
+    if (object[v]) {
+      res.concat(object[v], " ");
+    }
+  });
+  return res;
+};
+
+/**
+ * it adds the field requiring some manipulation to the payload
+ * @param {*} message
+ * @param {*} payload
+ */
+const refinePayload = (message, payload) => {
+  const app = message.context?.app?.name + message.context?.app?.build;
```

Comment:
If either of `name` or `build` is null/undefined , `app` value will result to NaN.

Reply:
Yeah it might be the case but while assigning the payload I am already removing those values usinhg `removeUndefinedAndNullValues()`.

Reply:
if we only have `message.context.app.name` then ideally we should send this .
but due to string concatenation of null/undefined 
const app = message.context?.app?.name + message.context?.app?.build;
`app` will be remove from final payload.